Source/UI/BillingForm/Default Implementation/Elements/DefaultBorderStyle.swift

```diff
@@ -1,11 +1,27 @@
 import UIKit
 
-struct DefaultBorderStyle: ElementBorderStyle {
-    var cornerRadius: CGFloat = Constants.Style.BorderStyle.cornerRadius
-    var borderWidth: CGFloat = Constants.Style.BorderStyle.borderWidth
-    var normalColor: UIColor = FramesUIStyle.Color.borderPrimary
-    var focusColor: UIColor = FramesUIStyle.Color.borderActive
-    var errorColor: UIColor = FramesUIStyle.Color.borderError
-    var edges: UIRectEdge? = .all
-    var corners: UIRectCorner? = .allCorners
+public struct DefaultBorderStyle: ElementBorderStyle {
+    public var cornerRadius: CGFloat = Constants.Style.BorderStyle.cornerRadius
+    public var borderWidth: CGFloat = Constants.Style.BorderStyle.borderWidth
+    public var normalColor: UIColor = FramesUIStyle.Color.borderPrimary
+    public var focusColor: UIColor = FramesUIStyle.Color.borderActive
+    public var errorColor: UIColor = FramesUIStyle.Color.borderError
+    public var edges: UIRectEdge? = .all
+    public var corners: UIRectCorner? = .allCorners
```

Comment:
Do we need to update any UI tests after adding the support for borders?

Reply:
UI tests validate that a component exists and is accessible, as if you were to tap it.

I think you are referring to snapshot tests, which we have not implemented and are currently not using in our SDK or sample application.